runtime/reference/migration_guide.md

```diff
@@ -255,7 +255,7 @@ See [deno#9795][deno#9795] for details.
 
 Use the `.close()` method on the resource instead.
 
-<a id="#rid">test</a>
+<a href="#rid" id="#rid">test</a>
```

Comment:
Looking at it again:

- The `id` attribute should not start with `#` as that is implied.
- Is it correct if we have a link that references itself?

Reply:
no, that should have been a # on the href and not on the id, clearly I am losing my mind.